CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Release Notes
+- Added a tag to published contact lists to help us detect the source of lost contact lists. [#51](https://github.com/planetary-social/cleanstr/issues/51)
```

Comment:
```suggestion
- Added a tag to published contact lists to help us detect the source of lost contact lists. [cleanstr#51](https://github.com/planetary-social/cleanstr/issues/51)
```
GitHub does this so it seems good for us to do it, too, whenever the issue is in a different repo.